File: services/payment/src/main/java/com/example/iter/payment/dto/toss/TossWebhookData.java
package com.example.iter.payment.dto.toss;

// paymentKey만 있으면 TossPaymentClient.getPayment()로 재조회해서 검증할 수 있으므로 나머지는 안 받는다.
public record TossWebhookData(String paymentKey, String orderId, String status) {
}

File: services/payment/src/main/kotlin/com/example/iter/payment/dto/toss/TossWebhookData.kt
package com.example.iter.payment.dto.toss

// paymentKey만 있으면 TossPaymentClient.getPayment()로 재조회해서 검증할 수 있으므로 나머지는 안 받는다.
@JvmRecord
data class TossWebhookData(val paymentKey: String, val orderId: String, val status: String)
